how-tos/rails/rake-tasks/readme.md

```diff
@@ -0,0 +1,65 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10899314/63829980-8188-11e5-8cdd-4ded5bcb6e36.png" height="60"> Custom Rake Tasks
+
+<img src="http://cdn.meme.am/instances/400x/58907263.jpg">
+
+## Why?
+
+* Creating custom rake tasks help companies and developers automate redundant activities that are needed for the maintenance of a website.
+
+* These tasks can be run with a rake command allow for the website to be optimized more effectively in production.
+
+## How?
+
+To check all of the rake tasks available to you in the Terminal:
+
+```cli
+rake -T
+```
+
+To create a custom task, you can go into your Terminal and run:
+
+```cli
+touch lib/tasks/fun_task.rake
+```
+
+**Note: You will want to name the file with .rake in order to allow Rails to look for it. However, you will be able to write all of your code using standard Ruby conventions.**
+
+This will allow you to house all of your tasks in the
+
+* A description
+* The name that identifies the task
+* The code to be executed by the task
+
+```rb
+desc "Rocky looks for Adrian after facing Creed"
+task :yo_adrian => :environment do
+  puts "Yo Adrian! I did it!!!"
+end
+```
+
+To run this task you can simply type:
+
+```cli
+rake yo_adrian
+```
+
+which should output:
+
+```cli
+Yo Adrian! I did it!!!
+```
+
+You can add as many tasks as you want into the same file. Doing so will help with your organization of the code.
+
+## Demo
+
+Note: Feel free to watch a demo of how to implement a custom rake task. The resources below will be a definitive guide for you as you begin to construct your own tasks in your applications.
+
+**Let's image that we have a database of 100 users that need to be reformatted on a daily basis. We need to create a task that will do this for us.**
```

Comment:
The use of "reformatted" seems ambiguous. I think we talked about calculating some sort of statistic about users like "average", then storing that in a column on the users table. I would just explain that here to be more clear.